Makefile

```diff
@@ -29,3 +29,20 @@ install:
 
 install-dev:
 	pip install -r requirements-dev.txt
+
+package:
```

Comment:
Is `package` a usual name for this target? Isn't something like `dist` very common?

Reply:
Sounds good as well